packages/datagateway-download/src/downloadApi.ts

```diff
@@ -293,6 +320,72 @@ export const downloadDeleted: (
     });
 };
 
+export const adminDownloadDeleted: (
+  downloadId: number,
+  deleted: boolean,
+  settings: {
+    facilityName: string;
+    downloadApiUrl: string;
+  }
+) => Promise<void> = (
+  downloadId: number,
+  deleted: boolean,
+  settings: {
+    facilityName: string;
+    downloadApiUrl: string;
+  }
+) => {
+  const params = new URLSearchParams();
+  params.append('facilityName', settings.facilityName);
+  params.append('sessionId', readSciGatewayToken().sessionId || '');
+  params.append('value', JSON.stringify(deleted));
+
+  return axios
+    .put(
+      `${settings.downloadApiUrl}/admin/download/${downloadId}/isDeleted`,
+      params
+    )
+    .then(() => {
+      // do nothing
+    })
```

Comment:
I think you don't technically need the `then()` here but perhaps it's good to keep in to make it explicit that we aren't doing anything with the response

Reply:
I agree. Other functions to the same so I will keep it as it is.